app/src/main/java/mozilla/lockbox/view/EditItemFragment.kt

```diff
@@ -49,48 +82,152 @@ class EditItemFragment : BackableFragment(), EditItemDetailView {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         setupToolbar(view.toolbar)
-    }
+        setTextWatcher(view)
+        setKeyboardFocus(view)
```

Comment:
Nit: use `setupX(view)` instead of `setX(view)`. 

If you're setting X, this would be analagous to `this.X = view`, which is not what you're doing.